viewer/.eslintrc.js

```diff
@@ -35,4 +35,10 @@ module.exports = {
       },
     },
   },
+  overrides: [{
+    files: ['*.spec.ts'],
+    env: {
+      jest: true,
+    },
+  }],
```

Comment:
This makes sure that when we are in a file that ends with `.spec.ts`, we can use the `describe`, `test`, `expect` etc. function without errors from the linter